infrastructure/modules/dynamodb/variables.tf

```diff
@@ -0,0 +1,14 @@
+# ==============================================================================
+# Mandatory variables
+
+variable "table_name" { description = "The table name of the DynamoDB" }
+
+# ==============================================================================
+# Default variables
+
+variable "hash_key" { default = "id" }
```

Comment:
We have, but at the stack level. Defaults are useful at the module level I think so that the calling stack doesn't have to worry about them. (less clutter at the stack level).  But happy to talk through...

Reply:
I don't think there's any harm in using defaults in a manifest that's this small.